### DISCUSSION TITLE
想要通过把目标路径作为参数启动可执行文件,让IDE直接打开对应工程目录

### DISCUSSION THREAD

**Discussion by Privauto** [rubickCenter/rubick#386] [Q&A]
像下图这种方式就可以实现，但部分IDE并不想做工程组和工程备注等等功能，我希望通过rubick实现一个工程快开的功能。
![image](https://github.com/user-attachments/assets/b204900f-236e-46c3-ba1a-d9351b159606)

**Comment by maplelost** [comment]
vscode快速打开项目
